python_tests/pytest_input_generators.py

```diff
@@ -522,30 +541,52 @@ def elementwise_binary_generator(
                 make_arg(rhs_shape, noncontiguous=True),
             )
 
+    # Create filtered special inputs for this operation's domain
+    lhs_domain_filter = partial(filter, partial(domain_filter, op.domain))
```

Comment:
This is also personal preference; take or leave:
```suggestion
    lhs_domain_filter = lambda xs: x for x in xs if domain_filter(op.domain, x)
```

Reply:
You can return a list here instead with
```python
lhs_domain_filter = lambda xs: [x for x in xs if domain_filter(op.domain, x)]
```
then won't need to explicitly do it for each call below.

Reply:
Lint complains about the lambda. https://www.flake8rules.com/rules/E731.html

I went with this:
```python
def filter_lhs_domain(xs):
  return [x for x in xs if domain_filter(op.domain, x)]
```

Reply:
Interesting! I didn't know named lambdas were an anti-pattern.